ncdjango/geoprocessing/params.py

```diff
@@ -402,6 +405,16 @@ def clean(self, value):
             raise ParameterNotValidError
 
         if source == 'service':
+            if '@' in value:
+                value, timestamp = value.split('@', 1)
+
+                try:
+                    service_time = datetime.datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=pytz.utc)
```

Comment:
Good point. I will replace this with something along the lines of `utcfromtimestamp(0) + timedelta(seconds=timestamp)` instead.